ci-cd/evaluate.py

```diff
@@ -15,49 +16,49 @@
 import torchvision.transforms as T
 from PIL import Image
 
-# (neptune) Fetch project
-project = neptune.get_project(
+# (Neptune) Set environment variable
+os.environ["NEPTUNE_PROJECT"] = "common/project-rl"
+
+# (Neptune) Fetch project
+project = neptune.init_project(
     api_token=os.getenv("NEPTUNE_API_TOKEN"),
```

Comment:
I think it would be nice to either pass both of them via API or set both of them as env variable.

Reply:
ah yes.. this is actually not even required as we read the token from the env by default